Detectors/Base/include/DetectorsBase/GeometryManagerParam.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef DETECTORS_BASE_INCLUDE_GEOMETRYMANAGERPARAM_H_
+#define DETECTORS_BASE_INCLUDE_GEOMETRYMANAGERPARAM_H_
+
+#include "CommonUtils/ConfigurableParam.h"
+#include "CommonUtils/ConfigurableParamHelper.h"
+
+namespace o2
+{
+
+struct GeometryManagerParam : public o2::conf::ConfigurableParamHelper<GeometryManagerParam> {
+  bool useParallelWorld = false;
+  bool addSensorToPW = false;
+  bool addMetalToPW = false;
+  bool addChipToPW = false;
+
+  O2ParamDef(GeometryManagerParam, "GeometryManagerParam");
```

Comment:
Same thought as above. This appears to be **too** generic, while some of the keys make sense only for ITS.

Would it be possible to move this to ITS and name it something like "ITSGeom" ...

Reply:
this is now moved to a configurable inside the ITS code. The general one only includes the parameter to fill the parallel geometry

Reply:
The code is still visible here: If it is moved ... probably this class can be deleted in the commit?

Reply:
I am not sure why this is not showing the updated version. In the latest commit, this class now contains the flags to enable the PW optimizations, while the ITS code is in another class under the ITS directory.

Reply:
I may have had a wrong eye here. All good for me.